goconst.sh

```diff
@@ -15,4 +15,12 @@
 
 
 GO111MODULE=off go get github.com/jgautheron/goconst/cmd/goconst
-goconst -min-occurrences=3 ./...
+
+if [[ $(goconst -min-occurrences=3 ./... | tee /dev/tty | wc -l) -ne 0 ]]
```

Comment:
what's the purpose of `tee /dev/tty` here?

Reply:
I'd like to see the actual output from `goconst`

Reply:
this is semi-standard way how to do it

Reply:
ah, got it, thanks, then looks good to me